lib/oli/delivery/sections/section_resource_depot.ex

```diff
@@ -178,12 +177,31 @@ defmodule Oli.Delivery.Sections.SectionResourceDepot do
 
     query =
       from sr in SectionResource,
-        where:
-          sr.section_id == ^section_id and sr.resource_type_id in [^page, ^container, ^objective],
+        where: sr.section_id == ^section_id,
+        where: sr.resource_type_id in [^page, ^container, ^objective],
         select: sr
 
     results = Repo.all(query)
 
     Depot.clear_and_set(@depot_desc, section_id, results)
   end
+
+  def fetch_recently_active_sections() do
```

Comment:
Should we move this section to the Sections context since it query information related to the sections, not the depot?
something like `fetch_sections_updated_since(time_ago)`
WDYT